fab

```diff
@@ -0,0 +1,19 @@
+#! /bin/bash
+
+ENV=.deployment_virtualenv
+# version of fusionbox-fabric-helpers to run
+HELPERS_COMMIT=f854d6a00
+HELPERS_COMMIT_FILE=$ENV/HELPERS_COMMIT
+
+if [ ! -d $ENV ]
+then
+  virtualenv -p `which python2` $ENV
+  touch $HELPERS_COMMIT_FILE
+fi
+source $ENV/bin/activate
+if [ "$HELPERS_COMMIT" != "`cat $HELPERS_COMMIT_FILE`" ]
```

Comment:
Can we use `$()` instead of back-ticks. This is not cargo cult, I find `$()` to be better style, back-ticks feels 70s.